### PR TITLE
psd-388-dotnet-parent-img

### PR DESCRIPTION
**Pull request by MJQ7**
https://eaflood.atlassian.net/browse/PSD-388

### CODE REVIEW

**Comment by MJQ7** [comment]
First daft of the netcore sdk file.

i have set the COMPlus_EnableDiagnostics in the entry point but i, not 100% this is correct but i think it is needed to run the container with a readonly file system.

File: ffc-netcore-parent/runtime/Dockerfile [hunk 1]
```diff
@@ -1,2 +1,7 @@
-ARG NETCORE_VERSION
-FROM mcr.microsoft.com/dotnet/core/aspnet:${NETCORE_VERSION}-alpine
```
Comment (johnwatson484) [inline ffc-netcore-parent/runtime/Dockerfile:2]: We should also include the .Net environment, ie `ENV ASPNETCORE_ENVIRONMENT=production`  the development Dockerfiles should have `ENV ASPNETCORE_ENVIRONMENT=development`

File: ffc-netcore-parent/sdk/Dockerfile [hunk 2]
```diff
@@ -1,2 +1,8 @@
 ARG NETCORE_VERSION=3.1
-FROM mcr.microsoft.com/dotnet/core/sdk:${NETCORE_VERSION}-alpine AS base
+FROM mcr.microsoft.com/dotnet/core/sdk:${NETCORE_VERSION}-alpine
```
Comment (johnwatson484) [inline ffc-netcore-parent/sdk/Dockerfile:2]: Should include `ENV ASPNETCORE_ENVIRONMENT=production`, the development equivalent should have development.

**Review by johnwatson484 — COMMENTED** [review]
Looks pretty much right but I think the remote debugger could be included in the SDK.  I've added more comments to our use of this parent https://github.com/DEFRA/ffc-demo-payment-service-core/pull/26

I've mentioned that we're not using the development images and on reflection do we really even need them?  Given the development and running stage is already split over the SDK and runtime image?  Could we just delete them?

**Review by johnwatson484 — APPROVED** [review]
LGTM

**Pull request merged**